README.md

```diff
@@ -209,6 +209,10 @@ pager: true
 width: 80
 # show all files, including hidden and ignored.
 all: false
+# show line numbers (TUI-mode only)
+showLineNumbers: false
+# preserve newlines in the output
+preserveNewLines: false
```

Comment:
I just wonder if it should be `showLineNumbers ` or `show_line_numbers `.

`preserveNewLines` is already camel case, though, so changing that would be a breaking change.

Other than this small decision, looks good to me.